setup.py

```diff
@@ -25,7 +22,6 @@
     entry_points={'console_scripts': ['pyxrf = pyxrf.gui:run']},
     package_data={'pyxrf.view': ['*.enaml'], 'configs': ['*.json']},
     include_package_data=True,
-    install_requires=['scikit-xray', 'matplotlib', 'enaml', 'six', 'numpy'],
```

Comment:
Well, install_requires should match the build time deps of the conda recipe. I'll fix that now